### ISSUE TITLE
FES vendor dashboard links missing

### ISSUE THREAD

**Issue by kiarashi** [RebelCode/edd-bookings#36]
Not sure if it's just me, but I noticed:

_?task=bookings-calendar_ and _?task=bookings_ are no longer working within the FES vendor dashboard (and EDD 2.8+) The usual links don't show up inside the vendor menu either (tested with twenty sixteen wp default theme).

**Comment by mecha** [comment]
@kiarashi We are not aware of such a problem. The _"Bookings"_ and _"Calendar"_ should be visible and functional in the FES vendor dashboard menu. Can you kindly check what version of EDD Bookings you are using from within the _Plugins_ page of your WordPress site?

**Comment by kiarashi** [comment]
It's version 2.2.0. 
We already contacted our customer (who sent us the extension) for the latest version to see if the problem remains or not. But based on the changelog, our issue hasn't been reported yet.

We used Twenty Eleven and only three plugins are activated: EDD, FES, Bookings.
Neither the links nor the pages show up inside the fes dashboard menu. 

If needed I can create a quick video snapshot

**Comment by mecha** [comment]
You're right - there don't seem to be any changes made to the FES vendor dashboard from our part.

However, I still recommend using the latest version, in the event that something else was changed in the intermediate versions that affected the integration with FES.

Given that we are unable to reproduce this on our sites, it makes most sense to assume that the cause of this problem can be found somewhere in the difference between our sites. Your currently installed versions of EDD and FES would help us to make our sites match yours as much as possible and hopefully narrow down the cause.

**Comment by kiarashi** [comment]
Here's a video preview: 
http://www.mediafire.com/file/aapnhbn1ebam8cj/bookings-issue.zip

Hope you're able to replicate the issue.

**Comment by mecha** [comment]
Ok, I've managed to replicate the problem using the same plugins that you have shown in the video, with the exact same versions (EDD 2.8.8, FES 2.5.8, Booking 2.0.0).

However, upon updating to the latest version of EDD Bookings, the vendor dashboard menus appeared. I urge you to do the same.
